operator/controllers/seldondeployment_controller.go

```diff
@@ -732,8 +735,9 @@ func createContainerService(deploy *appsv1.Deployment,
 			SessionAffinity: corev1.ServiceAffinityNone,
 		},
 	}
-	addLabelsToService(svc, pu, p)
-	addLabelsToDeployment(deploy, containerServiceKey, containerServiceValue)
+	deploy.ObjectMeta.Labels[containerServiceKey] = containerServiceValue
```

Comment:
Do we need to add a label to any service here? Just wondering as before we were also calling a `addLabelsToService` method.

Reply:
`addLabelsToService` doesn't add the `containerServiceKey` or `containerServiceValue` labels. Therefore when moving the `addLabelsTo...` functions up a level to the `createComponents` method I didn't add `svc.ObjectMeta.Labels[containerServiceKey] = containerServiceValue`

Perhaps this does need to be added? I'm not sure as it is not part of the set of labels that I am dealing with in this PR.